packages/core/src/managed/overview/PreviewEnvironments.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { EnvironmentOverview } from './EnvironmentOverview';
+import { EnvironmentsRender, OrderedEnvironments } from '../environmentBaseElements/EnvironmentsRender';
+import { QueryEnvironment } from './types';
+import { getDocsUrl } from '../utils/defaults';
+import { useLogEvent } from '../utils/logging';
+
+interface IPreviewEnvironmentsProps {
+  orderedEnvironments: OrderedEnvironments<QueryEnvironment>;
+  isConfigured?: boolean;
+}
+
+export const PreviewEnvironments = ({ orderedEnvironments, isConfigured }: IPreviewEnvironmentsProps) => {
+  const { environments: previewEnvironments, ...previewEnvironmentsProps } = orderedEnvironments;
+  const logClick = useLogEvent('PreviewEnvironments');
+  const docsLink = getDocsUrl('previewEnvironments');
+  return (
+    <>
+      {(Boolean(previewEnvironments.length) || isConfigured) && (
```

Comment:
Just out of curiosity, what's the advantage of writing it this way versus `(isConfigured && previewEnvironments.length > 0)` (which is more readable IMHO)?

Reply:
No advantage, just a personal perference. I don't mind changing though :)